scripts/dev-bundle-tool-package.js

```diff
@@ -54,7 +54,9 @@ var packageJson = {
     // https://github.com/williamwicks/node-eachline/pull/4
     eachline: "https://github.com/meteor/node-eachline/tarball/ff89722ff94e6b6a08652bf5f44c8fffea8a21da",
     cordova: "5.0.0",
-    pathwatcher: "4.1.0"
+    pathwatcher: "4.1.0",
+    babel: "5.5.1",
+    "source-map-support": "0.3.1"
```

Comment:
This is already a dependency, if you look up higher.

(And if you do need the upgrade, I'd suggest you keep the version in dev-bundle-server-package.js equal for simplicity.)